test/src/main/scala/scala/test/twitter_scrooge/BUILD

```diff
@@ -0,0 +1,100 @@
+load("//scala:scala.bzl",   "scala_binary", "scala_library")
+load("//twitter_scrooge:twitter_scrooge.bzl", "scrooge_scala_library")
+
+scrooge_scala_library(
+  name = "thrift1",
+  thrift_library = "//test/src/main/scala/scala/test/twitter_scrooge/thrift",
+  visibility = ["//visibility:public"],
+)
+
+scrooge_scala_library(
```

Comment:
But as the targets are traversed, they can label every source file with the target it comes from. This way in the final code gen target, we can just make sure that there are no identical files that have different content. Shouldn't that be sufficient? I like the conceptual simplicity of it right now and would prefer to keep it if possible.